src/Command/SelfUpdateCommand.php

```diff
@@ -274,4 +277,25 @@ private function cleanup(string $downloadedPhar): void
     {
         $this->filesystem->remove($downloadedPhar);
     }
+
+    private function updateComposer(): void
+    {
+        if ($this->input->getOption('dry-run')) {
+            return;
+        }
+
+        $composer = new Composer(
+            $this->downloader,
+            new Filesystem(),
+            $this->getWrappedOutput(),
+            $this->phpcqPath,
+            $this->config->getComposer(),
+            $this->findPhpCli()
+        );
+
+        if (! $composer->isBinaryAutoDiscovered()) {
```

Comment:
Is this enforced by code style? I seem to recall we do not space the `!` elsewhere.